src/stories/list.stories.js

```diff
@@ -50,11 +46,33 @@ export const ListA = () => {
 
     return (
         <StoriesContainer>
+            <p>Profile takes key/value - value as string or number only These are dumb componets</p>
             <Card title="Mentor List Example" onEdit={() => console.log("Edit is Clicked!")}>
                 <List items={items} >
 
-                    <h1>this is a child</h1>
-                    <div>So is this so intersting</div>
+                </List>
+            </Card>
+        </StoriesContainer>
+    );
+};
+
+
+
+// List takes children of type children
+export const ListGeneric = () => {
+
+
+    return (
+        <StoriesContainer>
+            <Card title="List Example" onEdit={() => console.log("Edit is Clicked!")}>
+                <List>
+                    <div> A list passed as children</div>
+                    <div>test</div>
+                    <div>test</div>
+                    <div>test</div>
+                    <div>test</div>
+                    <div>test</div>
```

Comment:
`List` can only accept `ListItem` (check `List.propTypes`).
Also Please use `items` to pass meaningful `type` and `value` like so:
`<ListItem type={items[0].type} value={items[0].value} />`
or use map.